core/src/main/java/tc/oc/pgm/consumable/ConsumableDefinition.java

```diff
@@ -0,0 +1,21 @@
+package tc.oc.pgm.consumable;
+
+import org.jetbrains.annotations.Nullable;
+import tc.oc.pgm.features.SelfIdentifyingFeatureDefinition;
+import tc.oc.pgm.kits.Kit;
+
+public class ConsumableDefinition extends SelfIdentifyingFeatureDefinition {
```

Comment:
generally for definitions prefer private fields with getters over simply protected fields